code/game/objects/structures/mouse_cage.dm

```diff
@@ -0,0 +1,123 @@
+
+/obj/item/critter_cage
+	name = "small cage"
+	desc = "A safe place where to keep tiny animals safe. Fit with a drinking bottle that can be refilled."
```

Comment:
as far as I can tell, nothing depletes the water? but maybe I'm not to that file yet

Reply:
For now only mice do anything while in the cages, I'll expand it later. They drink 1u of the reagents every Life() when in a cage.

Reply:
I feel like the drinking behavior should be on the cage. Perhaps even calling a simple_animal or even living level proc for caged behavior? The only downside to this is that it requires processing the cage.

Is it possible to distribute a virus or cure to the mouse by putting it in the bottle?

Reply:
Yes it is, though you can also just use a syringe on a mouse to inject them. You might want to put some virus food in the bottle.